test/Microsoft.Sbom.Parsers.Spdx22SbomParser.Tests/Parser/TestParser.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.Sbom.Contracts;
+using Microsoft.Sbom.Parsers.Spdx22SbomParser;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+namespace Microsoft.Sbom.Parser
+{
+    internal class TestParser
```

Comment:
This is a temporary testing class, once the final parser is written, I will update this to use that class instead.